src/components/counter/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { Component, createElement } from "react";
+
+class Counter extends Component {
```

Comment:
React adheres to the declarative programming paradigm so let's use function approach everywhere instead of classes 
https://react.dev/learn/reacting-to-input-with-state#how-declarative-ui-compares-to-imperative 

here is an example of using createcreateElement API with a functional approach 
https://react.dev/reference/react/createElement

Reply:
I will read the articles now.

Sorry, I though that this sentence means I should write with class components

The component should be written using EcmaScript classes syntax, the `render` method should use `React.createElement` API (without JSX).

Reply:
yeah, right, didn't notice, you can keep it like this then